test/systemtests/bgp_test.go

```diff
@@ -636,7 +636,7 @@ func (s *systemtestSuite) TestBgpMultiTrigger(c *C) {
 				s.CheckBgpConnectionForaNode(c, node.tbnode)
 			}
 		}
-		c.Assert(nodeToStop.startNetplugin("-vlan-if=eth2"), IsNil)
+		c.Assert(nodeToStop.startNetplugin(""), IsNil)
```

Comment:
The test suite already brings up the node with the necessary data interface configured in the test config file. There is no need to pass an vlan-if argument here. Verified that this is redundant. We shouldn't be doing the hard-coding here and the test config file handles this scenario.